starter-code/src/clue.js

```diff
@@ -1,94 +1,176 @@
-occupation:   Socialité
+var mrsPeacock = {
+  first_name: "Eleanor",
+  last_name: "Peacock",
+  color: "blue",
+  description:
+    "She is from a wealthy family and uses her status and money to earn popularity",
+  age: 36,
+  image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+  occupation: "Socialité"
+};
 
-mrMustard
-first_name:   Jack
-last_name:    Mustard
-color:        yellow
-description:  He is a former football player who tries to get by on his former glory
-age:          62
-image:        https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-occupation:   Retired Football player
+var mrMustard = {
+  first_name: "Jack",
+  last_name: "Mustard",
+  color: "yellow",
+  description:
+    " He is a former football player who tries to get by on his former glory",
+  age: 62,
+  image: " https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+  occupation: "Retired Football player"
+};
 
 // Weapons
 
-name: rope        ---   weight: 10
+/* name: rope        ---   weight: 10
 name: knife       ---   weight: 8
 name: candlestick ---   weight: 2
 name: dumbbell    ---   weight: 30
 name: poison      ---   weight: 2
 name: axe         ---   weight: 15
 name: bat         ---   weight: 13
 name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+name: pistol      ---   weight: 20 */
 
 // Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
 
 // Characters Collection
-var charactersArray = [];
+var charactersArray = [
+  mrGreen,
+  drOrchid,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  mrMustard
+];
 
 // Rooms' Collection
-var roomsArray = [];
+var roomsArray = [
+  { name: "Dinning Room" },
+  { name: "Conservatory" },
+  { name: "Kitchen" },
+  { name: "Study" },
+  { name: "Library" },
+  { name: "Billiard Room" },
+  { name: "Lounge" },
+  { name: "Ballroom" },
+  { name: "Hall" },
+  { name: "Spa" },
+  { name: "Living Room" },
+  { name: "Observatory" },
+  { name: "Theater" },
+  { name: "Guest House" },
+  { name: "Patio" }
+];
 
 // Weapons Collection
-var weaponsArray = [];
+var weaponsArray = [
+  {
+    name: "rope",
+    weight: 10
+  },
+
+  {
+    name: "knife",
+    weight: 8
+  },
+  {
+    name: "candlestick",
+    weight: 2
+  },
+  {
+    name: "dumbbell",
+    weight: 30
+  },
+  {
+    name: "poison",
+    weight: 2
+  },
+  {
+    name: "axe",
+    weight: 15
+  },
+  {
+    name: "bat",
+    weight: 23
+  },
+  {
+    name: "trophy",
+    weight: 25
+  },
+  {
+    name: "pistol",
+    weight: 20
+  }
+];
+
+function randomSelector(array) {
+  return array[Math.floor(Math.random() * array.length)];
+}
+function pickMistery() {
+  var misteryEnvelope = [];
+  misteryEnvelope.push(randomSelector(charactersArray));
```

Comment:
Un detalle menor: aquí podéis hacer un solo push con las tres llamadas.

Reply:
`misteryEnvelope.push(randomSelector(charactersArray),randomSelector(weaponsArray),randomSelector(roomsArray));`